src/xenia/base/profiling.cc

```diff
@@ -37,10 +37,8 @@
 #include "third_party/microprofile/microprofileui.h"
 #endif  // XE_OPTION_PROFILING
 
-#if XE_OPTION_PROFILING_UI
```

Comment:
This flag doesn't really seem to be useful anymore - since you're removing it here, can you remove it elsewhere? Removing #ifs makes me happy :)